web/template/how_should_attorneys_make_decisions.gohtml

```diff
@@ -25,7 +25,7 @@
                   {{ tr .App "jointlyAndSeverallyHumanised" }}
                 </label>
 
-                <div id="decision-type-item-hint" class="govuk-radios__hint">
+                <div id="decision-type-item-hint" class="govuk-hint govuk-radios__hint">
```

Comment:
has someone asked off the ticket for this to be hint colour?

Reply:
Ah, my bad - I was grabbing the hint component from the design system and just assumed we'd missed the additional class. I'll remove.